### PR TITLE
Added support webpack

### PR DESCRIPTION
**Pull request by SuperPaintman**
Hi!
I added exports from module and import angular library from **CommonJS** / **AMD**.

```js
const angular = require('angular');
const ngInfiniteScroll = require('ng-infinite-scroll');

angular.module('App', [
  ngInfiniteScroll // <- exports string with module name
])
```

### CODE REVIEW

File: src/infinite-scroll.coffee [hunk 1]
```diff
@@ -1,208 +1,218 @@
...
+      if attrs.infiniteScrollImmediateCheck?
+        immediateCheck = scope.$eval(attrs.infiniteScrollImmediateCheck)
+
+      checkInterval = $interval (->
+        if immediateCheck
+          handler()
+        $interval.cancel checkInterval
+      )
+  ]
+  .name
```
Comment (graingert) [inline src/infinite-scroll.coffee]: missing new line

Comment (graingert) [inline src/infinite-scroll.coffee]: traditionally you do something like:
```
const MODULE_NAME = 'infinite-scroll';
angular.module(MODULE_NAME).directive(...).component(...); // etc...
return MODULE_NAME;
```

**Comment by SuperPaintman** [comment]
@graingert done

**Comment by graingert** [comment]
closing in favour of https://github.com/sroze/ngInfiniteScroll/pull/271

**Pull request closed without merging**